templates/scenarios/non-sso-tab-js/tabs/infra/azure.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "resourceBaseName": {
+        "value": "v3sample${{RESOURCE_SUFFIX}}"
```

Comment:
Update the resource base name by "tab${{RESOURCE_SUFFIX}}"

Reply:
sry for missing here. has been fixed.